vendor/k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/stats/types.go

```diff
@@ -168,7 +168,7 @@ type MemoryStats struct {
 	// dirty memory, and kernel memory. WorkingSetBytes is <= UsageBytes
 	// +optional
 	WorkingSetBytes *uint64 `json:"workingSetBytes,omitempty"`
-	// The amount of anonymous and swap cache memory (includes transparent
+	// The amount of anonymous memory (includes transparent
```

Comment:
@andyxning thanks,

> Why would you change this vendor file without updating godeps.

Do you know who can verify the meaning of `rss` here ?

If I'm wrong in interpreting what `rss` mean ( if it include the cache ) than this pull request is wrong, and will not scratch my itch / BZ  :-)

> you should sign the CLA. :)

@simon3z @DirectXMan12 am I allowed to sign CLA ?